conftest.py

```diff
@@ -21,7 +21,7 @@ def pytest_addoption(parser):
         help="If provided, will not annihilate the data generated.")
 
 
-@pytest.fixture(scope="module")
+@pytest.fixture()
 def workspace(request):
     # TODO: Use pytest.config.option('--workspace')
     test_workspace = tempfile.mkdtemp()
```

Comment:
if you change `testbench_performance.py:L34 to `logging.DEBUG` instead of `INFO`, it'll print the dirs used, and you can check that they (a) are new each call and (b) aren't there at the end.